housekeeper/cli/delete.py

```diff
@@ -99,54 +101,79 @@ def version_cmd(context, bundle_name, version_id, yes):
 @click.pass_context
 def files_cmd(context, yes, tag, bundle_name, before, notondisk, list_files, list_files_verbose):
     """Delete files based on tags."""
+
+    validate_delete_options(tag=tag, bundle_name=bundle_name)
+    before_date = parse_date(before) if before else None
     store = context.obj["store"]
-    file_objs = []
-    if not (tag or bundle_name):
-        LOG.info("Please specify a bundle or a tag")
-        raise click.Abort
 
     if bundle_name:
-        bundle_obj = store.bundle(bundle_name)
-        if bundle_obj is None:
-            LOG.warning("Bundle not found")
-            raise click.Abort
+        validate_bundle_exists(store, bundle_name)
 
-    query = store.files_before(bundle=bundle_name, tags=tag, before=before)
+    files = store.get_files_before(bundle=bundle_name, tags=tag, before_date=before_date)
```

Comment:
Fix function argument name to reflect input.

```suggestion
    files = store.get_files_before(bundle_name=bundle_name, tags=tag, before_date=before_date)
```